src/Fantomas/CodePrinter.fs

```diff
@@ -1089,14 +1090,14 @@ and genExpr astContext synExpr ctx =
                     +> sepSpace
                     +> col sepSpace args (genExpr astContext)
                     +> sepSpace
-                    +> genArrayOrList astContext isArray openingTokenRange children closingTokenRange
+                    +> genArrayOrList astContext isArray openingTokenRange children closingTokenRange ctx.Config.MaxElmishWidth
```

Comment:
Interesting to see that this just works. Good thinking.

Reply:
Yeah we'll see how far I get re-using that function.  Might end up diverging in the end.